.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an idea/feature/enhancement for AWS SAM Build Images
+title: "Feature request: TITLE"
+labels: ['type/feature', 'stage/needs-triage']
+assignees: ''
+
+---
+
+<!-- Make sure we don't have an existing Issue for that feature request (open or closed). -->
+
+### Describe your idea/feature/enhancement
+
+Provide a clear description.
```

Comment:
Shouldn't this be HTML comment as well like in `bug_report.md`?

Reply:
Probably, but in this PR I mainly want to keep consistency across repos (as these issue templates are 99% copy-pasted across repos), so don't want to only have it differently here.  We could have follow-up PRs to update it in all the repos if necessary.